client/src/app/components/stardust/Transaction.scss

```diff
@@ -13,6 +13,22 @@
     display: none;
   }
 
+  .date {
```

Comment:
I don't like adding rules to an un-related scss file ? Why can't we make a scss file specific for Activity.tsx ?

Reply:
Done.